HACKING.md

```diff
@@ -21,59 +21,60 @@ Setup
 
 Set a development setup with [`musicbrainz-docker`](https://github.com/metabrainz/musicbrainz-docker).
 
-This can be used to run any released or development version of
-MusicBrainz Server (MBS -
+This can be used to run any released or development versions of:
+* MusicBrainz Server (MBS -
 [repository](https://github.com/metabrainz/musicbrainz-server/),
 [search-related tickets](https://tickets.metabrainz.org/issues/?jql=project%20%3D%20MBS%20AND%20component%20%3D%20Search%20AND%20status%20!%3D%20Closed)
 ),
-Search Index Rebuilder (SIR -
+* Search Index Rebuilder (SIR -
 [repository](https://github.com/metabrainz/sir/),
 [tickets](https://tickets.metabrainz.org/issues/?jql=project%20%3D%20SEARCH%20AND%20component%20%3D%20Indexer%20AND%20status%20!%3D%20Closed)
 ),
-and a Solr search server (MB Solr -
+* MusicBrainz Solr search server (MB Solr -
 [repository](https://github.com/metabrainz/mb-solr/),
 [tickets](https://tickets.metabrainz.org/issues/?jql=project%20%3D%20SEARCH%20AND%20component%20%3D%20%22Response%20Writer%22%20AND%20status%20!%3D%20Closed)
 )
 which comes with a Solr schema (MBSSSSS -
 [repository](https://github.com/metabrainz/mbsssss/),
 [tickets](https://tickets.metabrainz.org/issues/?jql=project%20%3D%20SEARCH%20AND%20component%20%3D%20Schema%20AND%20status%20!%3D%20Closed)
 ).
+
 These versions of MBS, SIR and MB Solr should
 (preferably for development, necessarily for production)
-conform to the same version of Relax NG schema (MMD -
+conform to the same version of MusicBrainz Relax NG schema (MMD -
 [repository](https://github.com/metabrainz/mmd-schema)
-) as it may just not work together otherwise.
+) as they may just not work together otherwise.
 
-See also [MusicBrainz search architecture](https://musicbrainz.org/doc/Development/Search_Architecture) for a broader overview.
+See also [MusicBrainz search architecture](https://musicbrainz.org/doc/Development/Search_Architecture) for a broader overview of the whole setup.
 
 Solr logs
 ---------
 
-Logs can be useful when making a search query with MBS or when indexing documents with SIR.
+Logs can be used when making a search query with MBS and when indexing documents with SIR.
```

Comment:
I’m not sure it’s clearer. Logs can be used anytime. They are useful to when doing one and/or the other.

Reply:
Tried to improve it with commit 4688383